ftprime/recomb_collector.py

```diff
@@ -37,8 +37,13 @@ def __init__(self, nsamples, generations, N, ancestor_age, length, locus_positio
         self.length=length
         self.locus_position=locus_position
         self.last_child=-1
+        self.time = float(generations) + 1
 
         self.args=ARGrecorder()
+
+        # TODO is this needed with refactoring in msprime to nodes/tables?
```

Comment:
regardless of whether it is needed, we got to start counting somewhere, and this is a good, sensible place to start